core/templates/dev/head/pages/exploration_player/PredictionAlgorithmRegistryService.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service for mapping algorithmId to PredictionAlgorithmService.
+ */
+
+ oppia.factory('PredictionAlgorithmRegistryService', [
+   'PredictionAlgorithmSampleService',
+   function(PredictionAlgorithmSampleService) {
+    /**
+    * This mapping needs to be updated whenever a new prediction service needs
+    * to be added for classification. The mapping is from algorithmId to a
+    * list of objects. The mapping should be of the type:
+    * {
+    * algorithmId: [{
```

Comment:
Also indent this entire comment block by 1 so that the lowermost asterisk lines up with the var just below it.